src/composables/useDevices.js

```diff
@@ -106,6 +109,15 @@ export function useDevices(video, initializeOnMounted) {
 
 		virtualBackground.value.connectTrackSink('default', videoTrackToStream.value, 'video')
 
+		const blurBackgroundEnabled = settingsStore.blurBackgroundEnabled
+		if (blurBackgroundEnabled) {
```

Comment:
```suggestion
		if (settingsStore.blurBackgroundEnabled) {
```

And I think, same here: MediaSettings is doing it on open, MediaDevicesPreview should also have this logic on mounted, but not the shared composable

Reply:
Okay, I apply the main blur setting on component level and not in composable (open for MediaSettings, and mount for MediaDevicesPreview)